patches/node/build_add_gn_build_files.patch

```diff
@@ -40,15 +40,7 @@ index 35ccd0487f20cece033d58827ecb7ed016908ee4..62cd49c6a87074912a1cb6792576c8d4
  
    # The location of OpenSSL - use the one from node's deps by default.
    node_openssl_path = "$node_path/deps/openssl"
-@@ -39,12 +39,15 @@ declare_args() {
-   # The variable is called "openssl" for parity with node's GYP build.
-   node_use_openssl = true
- 
-+  # Build node with SQLite support.
-+  node_use_sqlite = true
```

Comment:
Feels like incorrect patch change, refs https://github.com/electron/electron/blob/main/patches/node/build_add_gn_build_files.patch.

Reply:
Hmm, this may have been a bad rebase with main - thanks, now fixed